frontend/src/api/k8s/inferenceServices.ts

```diff
@@ -200,10 +200,19 @@ export const assembleInferenceService = (
     data.isKServeRawDeployment,
   );
 
-  // Only add resources for KServe, but not tolerations and nodeSelector
   if (!isModelMesh && podSpecOptions) {
-    const { resources } = podSpecOptions;
-
+    const { tolerations, resources, nodeSelector } = podSpecOptions;
+    const isLegacyHardwareProfile =
+      podSpecOptions.selectedHardwareProfile &&
+      !podSpecOptions.selectedHardwareProfile.metadata.uid;
```

Comment:
i guess this isnt really legacy profile but rather if there is no selected hardware profile (so either legacy hwp or accelerator profile)

im not sure this logic supports the accelerator profile feature still?